ui/app/clinical/common/services/treatmentService.js

```diff
@@ -135,13 +135,34 @@ angular.module('bahmni.clinical')
             return deferred.promise;
         };
 
+        var getMessageForPrescription = function (patientData, drugOrderSection) {
```

Comment:
We should use a translation string template, not use concatenation to construct the message.

Reply:
Also, this code I think can be its own javascript class, with a getPrescriptionAsString( ) method, and therefore can also have its own unit test cases, to validate the conversions.

Reply:
Each drug order can have a getDrugOrderAsString( ) and return the string representation. isn't it?